### ISSUE TITLE
SR: Settings - Preferences: The Language options are announced without a role

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74836]
## Action Performed
Prerequisite:
- User is logged in

Steps:
1. Using JAWS + Chrome, open the URL: https://new.expensify.com/
2. Tab to focus on the "My Settings" button and press Enter to activate it
3. Once the My Settings menu is displayed, tab to focus on the "Preferences" menu item and press Enter to activate it
4. Tab to focus on the "Language" button and press Enter to expand it
5. Use the tab key to focus on any of the required language
6. Observe the screen reader announcement

Other Occurrences:
1. On Settings - Preferences - Priority mode
2. On Settings - Profile - Address - Country
3. On Workspace - Rules - Cash Expense Default
4. On Workspace - Company cards: the options of countries
5. On Workspace - Company cards: the options of banks
6. On Workspace - Company cards: the options of connect options
7. On Workspace - Create Workspace - Default Currency
8. On Create Report - Change payment currency
9. On Wallet - Add bank account
10. On Paid Expense details flow

## Expected Result
The language options should be announced with an actionable role such button by the screen reader.

## Actual Result
When navigating through the available Language options (e.g., English), each option is announced without any actionable role by the screen reader. When the controls are focused the screen reader announces only the label of the control as "english". As a result, users cannot determine whether these options are actionable or not or how to interact with them.

HTML:
<div class="css-g5y9jx" style="flex-direction: column; position: absolute; width: 375px; left: 0px; top: 64px;"><div class="css-g5y9jx" data-component="OfflineWithFeedback" data-element="View" data-source-file="OfflineWithFeedback.tsx"><div class="css-g5y9jx"><div class="css-g5y9jx" data-component="AnimatedComponent" data-element="Component" data-source-file="createAnimatedComponent.js" style="opacity: 1;"><div aria-label="English" role="presentation" data-tag="pressable" data-selection-scrapper-hidden-element="true" data-inner-box-shadow-element="false" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21" id="en" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style=""><div class="css-g5y9jx r-13awgt0 r-1wtj0ep r-1awozwy r-18u37iz r-b5h31w r-1ah4tor r-9ffhgg r-lrvibr r-adacv r-kzbkwu r-ttdzmv" data-testid="base-list-item-en" style=""><div class="css-g5y9jx r-13awgt0 r-1habvwh"><div dir="auto" class="css-146c3p1 r-dnmrzs r-1udbk01 r-1iln25a" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; min-height: 20px; line-height: 20px; white-space: pre; text-decoration-line: none; overflow: hidden;">English</div></div><div class="css-g5y9jx r-18u37iz r-1awozwy r-19u6a5r"><div class="css-g5y9jx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#03D47C"><path d="M17.2 3.4c.9.7 1.1 1.9.4 2.8L8.9 18l-6.3-5.7c-.8-.7-.9-2-.1-2.8.7-.8 2-.9 2.8-.1l3 2.7 6.1-8.2c.6-1 1.9-1.2 2.8-.5" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div></div></div></div></div></div></div></div>

## Area issue was found
Settings-Preferences

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Ensure the language options are announced with actionable role by the screen reader. Change the "role=presentation" to role="button" for every control.

Refer:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/button_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/presentation_role

## Environment(s)
Windows 11 Home - Chrome

<img width="1927" height="970" alt="Image" src="https://github.com/user-attachments/assets/a06a197c-de6d-428d-9569-08bc3db7022c" />

https://github.com/user-attachments/assets/0d25ca70-5705-43ab-abf6-b023b1a19e7e

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995880
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
Accessibility, addressing as part of a project.

**Comment by rushatgabhane** [comment]
cool, closing as fixed
